src/abstractdataframe/abstractdataframe.jl

```diff
@@ -884,13 +887,17 @@ nonunique(df::AbstractDataFrame, cols::Any) = nonunique(df[cols])
 Base.unique!(df::AbstractDataFrame) = deleterows!(df, findall(nonunique(df)))
 Base.unique!(df::AbstractDataFrame, cols::AbstractVector) =
     deleterows!(df, findall(nonunique(df, cols)))
+Base.unique!(df::AbstractDataFrame, cols::Regex) =
```

Comment:
Can be merged with previous and next method.

Reply:
Merging them would lead to method ambiguity with methods defined for `unique!` in Base. The problematic method there is `unique!(f, A::AbstractArray{T,1} where T)`.